utils.go

```diff
@@ -0,0 +1,18 @@
+package main
```

Comment:
@bfallik In the future I would appreciate it if you would elaborate more in your comments instead of indicating that something is "wrong", as this wastes my time and Clypd's dollars.